modules/mod_video/support/z_video_convert.erl

```diff
@@ -100,8 +101,9 @@ do_convert(QueuePath, State) ->
             lager:warning("ffmpeg conversion error on ~p: ~p", [State#state.id, Error]),
             insert_broken(State)
     end,
-    mod_signal:emit({medium_update, [{id,State#state.id}]}, Context).
+    mod_signal:emit({medium_update, [{id, State#state.id}]}, Context).
 
+-spec insert_movie(string(), #state{}) -> {ok, m_rsc:resource_id()} | {error, atom()}.
```

Comment:
The spec in line 106 uses a record, please define a type for the record and use that instead.